src/test/java/com/google/api/codegen/testdata/discoveries/go/go_logging.v2beta1.json.baseline

```diff
@@ -196,27 +288,49 @@ import (
 
 func main() {
   ctx := context.Background()
+
+  // Authentication is provided by the `gcloud` tool when running locally
+  // and by built-in service accounts when running on GAE, GCE, or GKE.
+  // For more information, see
+  // https://developers.google.com/identity/protocols/application-default-credentials
   hc, err := google.DefaultClient(ctx, logging.CloudPlatformScope)
   if err != nil {
     // TODO: Handle error.
+    log.Fatal(err)
   }
-  c, err := logging.New(hc)
+
+  // Construct the logging service object for interacting with the API.
+  // You can browse other available API services and versions at
+  // https://github.com/google/google-api-go-client
+  service, err := logging.New(hc)
   if err != nil {
     // TODO: Handle error.
+    log.Fatal(err)
   }
 
+  // TODO: Change placeholders below to desired parameter values for the `Create` call:
+
   // The resource name of the project in which to create the metric. Example: `"projects/my-project-id"`.
   // The new metric must be provided in the request.
-  projectName := "projects/{MY-PROJECT}" // TODO: Update placeholder value.
```

Comment:
This seemed a lot nicer before.

The TODO above has a lot of nonsense. Of course it's for the Create call. That's the point of the entire piece of code.